specutils/spectra/spectrum1d.py

```diff
@@ -165,6 +182,53 @@ def bin_edges(self):
     def shape(self):
         return self.flux.shape
 
+    @property
+    def redshift(self):
+        """
+        The redshift(s) of the objects represented by this spectrum.  May be
+        scalar (if this spectrum's ``flux`` is 1D) or vector.  Note that
+        the concept of "redshift of a spectrum" can be ambiguous, so the
+        interpretation is set to some extent by either the user, or operations
+        (like template fitting) that set this attribute when they are run on
+        a spectrum.
+        """
+        return self._radial_velocity/cnst.c
+    @redshift.setter
+    def redshift(self, val):
+        if val is None:
+            self._radial_velocity = None
+        else:
+            self.radial_velocity = val * cnst.c
```

Comment:
This assumption is really only true for small redshifts. Should we use the stricter definition?